### PR TITLE
preparing for release 0.1.9

### PR DESCRIPTION
**Pull request by likawind**
## Checklist before requesting a review
- [ ] I have created a descriptive PR title. The PR title should complete the sentence "This PR...".
- [ ] I have performed a self-review of my code.
- [ ] I have included a small demo of the changes. For the UI, this would be a screenshot or a Loom video.
- [ ] If this is a new feature, I have added unit tests and integration tests.
- [ ] I have run the integration tests locally and they are passing.
- [ ] I have run the linter script locally (See `python3 scripts/run_linters.py -h` for usage).
- [ ] All features on the UI continue to work correctly.
- [ ] Added one of the following CI labels:
    - `run_integration_test`: Runs integration tests
    - `skip_integration_test`: Skips integration tests (Should be used when changes are ONLY documentation/UI)

### CODE REVIEW

**Comment by likawind** [comment]
@vsreekanti I noticed that we name the branch differently, I can help merging changelogs in `release_019` to this branch whenever you are ready

**Comment by cw75** [comment]
@likawind I think we can merge `release_019`'s changelog to here. Could you double check if the changelog is up-to-date? Especially see if the following features are merged:

- Cascading workflow triggers
- Eunice's PRs on cleaning up workflow table appearance
- Persist the number of rows shown per-page on metadata views between page refreshes
- Avoiding usage stats collection in dev mode
- Hiding previews when artifacts are canceled
- Hiding parameters in the status bar
- Improve spacing on workflow table view

If so, add then to the changelog. Thx!

**Review by likawind — COMMENTED** [review]
@cw75 @vsreekanti could you help checking the amended changelogs?

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,33 @@
...
 
+## 0.1.9
+Released on Januay 9, 2023.
+
+### Key Features
+* As of this release, Aqueduct has usage tracking. Usage tracking is fully
+    anonymized and captures API routes, performance data, and error rates
+    without revealing any specifics of your machine or deployment. For more
+    details, check out our [documentation](https://docs.aqueducthq.com/usage).
+* We now support cascading workflow triggers. You can specify that in our [python SDK](https://github.com/aqueducthq/aqueduct/blob/main/sdk/aqueduct/client.py#L373).
```
Comment (cw75) [inline CHANGELOG.md:11]: @likawind maybe add a sentence explaining what it means. Sth like "We now support cascading workflow triggers, meaning that a workflow can trigger another one at the end of its execution."

**Review by cw75 — APPROVED** [review]

**Pull request merged**